.github/workflows/test.yml

```diff
@@ -8,15 +8,8 @@ jobs:
     steps:
       - name: Checkout Code
         uses: actions/checkout@v4
-      - name: Install Node.js
-        uses: actions/setup-node@v4
-        with:
-          node-version: 18
-      - name: Install Dependencies
-        run: npm ci
-        working-directory: ./backend/main
       - name: Run Tests
-        run: npm test
+        run: docker compose up app --exit-code-from app
```

Comment:
This option is needed to stop running docker after a test is finished.

https://docs.docker.com/engine/reference/commandline/compose_up/